pkg/model/fileshare.go

```diff
@@ -39,6 +39,10 @@ type FileShareAclSpec struct {
 	// The description of the fileshare acl.
 	Description string `json:"description,omitempty"`
 
+	// The status of the fileshareacl.
+	// One of: "available", "error" etc.
+	Status string `json:"status,omitempty"`
```

Comment:
If you want to add the Status field, please also change swagger.yaml

Reply:
ok i will raise another PR to correct swagger.yaml